### PR TITLE
Caching fix

### LINKED ISSUE #5933: Reconsider font legibility on streak badges
Maybe reinstate the text shadow or change the font colour so it's easier to read.

https://oc.app/community/dgegb-daaaa-aaaar-arlhq-cai/channel/174681535352094284612163711348576298496/4379

*Update*: this is a caching issue. We need to understand what's going on there. The correct version _should_ be in cloudflare. Which means that some people just have the incorrect version stuck in their client cache. But that implies that the google workbox stale-while-revalidate strategy is simply not working.

### PR DESCRIPTION
**Pull request by julianjelfs**
fixes #5933

### CODE REVIEW

File: backend/canisters/storage_bucket/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- Add cacheable resource header ([#5947](https://github.com/open-chat-labs/open-chat/pull/5947))
```
Comment (megrogan) [inline backend/canisters/storage_bucket/CHANGELOG.md:11]: There is already a Changed section so no need to add a new one

Comment (julianjelfs) [reply backend/canisters/storage_bucket/CHANGELOG.md:11]: There are already _lots_ of changed sections. I don't really understand what this log is supposed to be telling us.

Comment (julianjelfs) [reply backend/canisters/storage_bucket/CHANGELOG.md:11]: Oh I think I get it. Release tags interspersed with changed, added etc

File: backend/canisters/storage_bucket/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
```
Comment (megrogan) [inline backend/canisters/storage_bucket/CHANGELOG.md:11]: Typically we put this at the bottom of the list but it doesn't really matter! Hamish might have rejected your PR though ;-)

**Review by megrogan — APPROVED** [review]

File: backend/canisters/storage_bucket/CHANGELOG.md (hunk 2 above)
Comment (megrogan) [reply backend/canisters/storage_bucket/CHANGELOG.md:11]: Feel free to resolve this conversation!

**Pull request merged**